call_center.py

```diff
@@ -40,49 +41,98 @@
     Price - 22$
     '''
 
+menu_ld = [{'Traditional': 19}, {'Margherita': 22}, {'Pizza Bianca': 21},
+           {'Marinara': 17}, {'Havaiian': 25}, {'Bruschetta': 22}]
+
 all_stuff = ['Hanna', 'Greg', 'Michael', 'Chloe', 'Tyler', 'Ashley', 'Jessica']
 stuff = random.choice(all_stuff)
 
 greetings = ['Hi!', 'Hello!', 'Yo!', 'Sup?', 'Whazzup?', 'Howdy!', 'Hi there!']
 hi = random.choice(greetings)
 
 misunderstanding = [
-    'What do you mean? Does it means yes or no?',
+    'What do you mean? Does it mean yes or no?',
     'Could you please repeat that?',
-    'Yes or No? You can answer in russian if you want...',
     'Sorry, I dont understand. Do you mean yes?',
-    'Not so fast cowboy! You push the wrong button ~_~',
+    'Not so fast cowboy! You push a wrong button ~_~',
     'C\'mon dude! Do you want some pizza or not?'
 ]
 
 
 # CHAT STARTS HERE>>>>
 
+print("********************************************")
 print('''
 {} My name is {}! What do I call you?
 '''.format(hi, stuff))
 usr_name = input()
 
-# print('''
-# Ok {}! Looking for some classic italian pizza?'''.format(usr_name))
-
 print('''
-Ok {}! Looking for some classic italian pizza? [Y/n (Д/н)]:
+Nice to meet you, {}! Looking for some classic italian pizza? [Y/n]:
 '''.format(usr_name))
 
 while True:
     response = input().upper()
 
-    if re.match("[YД]", response):
-        print(menu_full)
+    if re.match("[Y]", response):
+        print()
         break
-    elif re.match("[NН]", response):
+    elif re.match("[N]", response):
         print()
```

Comment:
You have lots of `print()` across the code, maybe it would be a good idea to format string the way they contain all needed spaces if that's possible of course.

Reply:
I picked that up too )))